### PR TITLE
KOGITO-2460: Update to version 0.5.1

### PR DESCRIPTION
**Pull request by caponetto**
I've temporarily added the patternfly-base code while we don't have `@kogito-tooling/patternfly-base` ready for usage. 
I'll update this repo accordingly as soon as a new release of `@kogito-tooling/patternfly-base` is published.

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -11,9 +11,9 @@
     "target": "es5",
     "declaration": true,
     "sourceMap": true,
-    "noImplicitAny": true,
```
Comment (tiagobento) [inline tsconfig.json:14]: Any particular reasons to remove that? I guess not having implicit `any` types is supposed to be good?

Comment (caponetto) [reply tsconfig.json:14]: The reason for removing that was only to match what we currently have on `kogito-tooling` repo but I can put it back.

Comment (tiagobento) [reply tsconfig.json:14]: Oh..... I didn't know we removed that from `kogito-tooling`. We should probably put it back :D

File: packages/simple-react-editors/src/SimpleReactEditor.tsx [hunk 2]
```diff
@@ -41,36 +39,42 @@ export class SimpleReactEditor extends React.Component<Props, State> {
...
+  public async getContent(): Promise<string> {
+    return this.state.content;
   }
 
-  //saving triggers this method, so we also update the originalContent by calling `this.setContent`
-  public getContent() {
-    return this.setContent(this.state.content).then(() => this.state.content);
+  public isDirty(): boolean {
+    throw new Error("Method not implemented.");
   }
```
Comment (tiagobento) [inline packages/simple-react-editors/src/SimpleReactEditor.tsx:59]: I think we should deprecate this method on `kogito-tooling`. This is not used and we have a different way of identifying that a file has been edited.

Comment (caponetto) [reply packages/simple-react-editors/src/SimpleReactEditor.tsx:59]: +1

**Review by ederign — APPROVED** [review]

**Review by tiagobento — APPROVED** [review]
Let's create issues for the comments I posted. We can address them later.

**Pull request merged**